src/Fabulous/Cmd.fs

```diff
@@ -80,6 +117,9 @@ module Cmd =
 
             [ bind ]
 
+    //TODO what do I use this for? Compared to other OfAsync... modules?
```

Comment:
`OfAsyncWith` is the base for `OfAsync` and `OfAsyncImmediate`. By itself, it lets you choose how you want the async computations to be executed. But in practice, you would use `OfAsync` when you want the async comp. to be executed on the thread pool, or `OfAsyncImmediate` when you want the async comp. to be executed on the current thread.

Reply:
Not much point to use `OfAsyncWIth` directly since the only 2 options are `Async.Start` and `Async.StartImmediate` which are covered by `OfAsync` and `OfAsyncImmediate`.